### DISCUSSION TITLE
windows 运行 exe 文件如何隐藏窗口？

### DISCUSSION THREAD

**Discussion by SkysCrystal** [zdz/ServerStatus-Rust#251] [Q&A] [answered]
我使用了 windows 任务计划程序，运行 exe 并带参数，已成功连接服务端。
该如何隐藏程序运行时候的 cmd 窗口呢？求解答，谢谢！

**Comment by zdz** [comment, marked as answer]
https://www.google.com/search?q=NSSM+-csdn
